dataplane/worker.cpp

```diff
@@ -4025,11 +4041,18 @@ inline void cWorker::balancer_handle()
 
 			key.balancer_id = metadata->flow.data.balancer.id;
 			key.protocol = metadata->transport_headerType;
+			key.l3_balancing = service.flags & YANET_BALANCER_PURE_L3;
 			key.addr_type = 6;
 			memcpy(key.ip_source.bytes, ipv6Header->src_addr, 16);
 			memcpy(key.ip_destination.bytes, ipv6Header->dst_addr, 16);
 			key.port_source = 0;
 			key.port_destination = 0;
```

Comment:
if service requires pure L3 balancing, we leave ports fields with zeros, otherwise we miss session in balancer state in case of port changing. Hence we lose information about client_port and real_port in `yanet-cli balancer state`. Is this acceptable?

Reply:
I do not think we should show ports here as many real TCP sessions will be mapped into one `balancer session`